package/AUTHORS

```diff
@@ -125,7 +125,8 @@ Chronological list of authors
   - Lily Wang
   - Matthijs Tadema
   - Joao Miguel Correia Teixeira
-
+2020
+  - Charlie Cook
```

Comment:
Can you add an empty line between the end of the author list and the `External code` header please?